examples/timestamping/backend/src/lib.rs

```diff
@@ -77,3 +78,29 @@ impl Service for TimestampingService {
         TimestampingApi.wire(builder);
     }
 }
+
+impl Configure for TimestampingService {
+    type Params = Config;
+
+    fn verify_config(
+        &self,
+        _context: ExecutionContext<'_>,
+        params: Self::Params,
+    ) -> Result<(), ExecutionError> {
+        if params.time_service_name.is_empty() {
```

Comment:
I don't think this is a sufficient check. See [how it's checked above](https://github.com/exonum/exonum/blob/356b267d64ca2fe89e0cbd446fcdafd265fd7ec7/examples/timestamping/backend/src/lib.rs#L64).